data_validation/validation_builder.py

```diff
@@ -397,37 +380,21 @@ def get_target_query(self):
             "data_client": self.target_client,
             "schema_name": self.config_manager.target_schema,
             "table_name": self.config_manager.target_table,
+            "target_query": self.config_manager.target_query,
         }
+
         if self.validation_type == consts.CUSTOM_QUERY:
-            target_input_query = self.get_query_from_file(
-                self.config_manager.target_query_file[0]
+            table = clients.get_ibis_query(
+                self.target_client, self.config_manager.target_query
             )
-            if self.config_manager.custom_query_type == "row":
-                calculated_query = CustomQueryBuilder().compile_custom_query(
-                    target_input_query, target_config
-                )
-                query = self.target_client.sql(calculated_query)
-            elif self.config_manager.custom_query_type == "column":
-                target_aggregate_query = "SELECT "
-                for aggregate in self.config_manager.aggregates:
-                    target_aggregate_query += (
-                        CustomQueryBuilder().get_aggregation_query(
-                            aggregate.get("type"), aggregate.get("target_column")
-                        )
-                    )
-                target_aggregate_query = (
-                    CustomQueryBuilder().get_wrapper_aggregation_query(
-                        target_aggregate_query, target_input_query
-                    )
-                )
-                query = self.target_client.sql(target_aggregate_query)
-            else:
-                raise ValueError(
-                    "Expected custom query type to be column or row, got an unacceptable value. "
-                    f"Input custom query type: {self.config_manager.custom_query_type}"
-                )
         else:
-            query = self.target_builder.compile(self.validation_type, **target_config)
+            table = clients.get_ibis_table(
```

Comment:
line 361-364 in config_manager.py fetches the table object. Do we need this again here?

Reply:
Same as above

Reply:
Let's use the config_manager for this too - `table = self.config_manager.get_target_ibis_table()`

Reply:
Agreed. This is incorporated [here](https://github.com/GoogleCloudPlatform/professional-services-data-validator/pull/607/commits/f0eb38bd9e472989bcb8a8ed43a69350c1d28973)